src/ci/unset-versions.sh

```diff
@@ -7,7 +7,12 @@ set -ex
 
 SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
 GET_VERSION=${SCRIPT_DIR}/get-version.sh
```

Comment:
it looks like we don't need `GET_VERSION` and `VERSION` during unset. will remove it.

Reply:
removed